front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,347 @@
+                <View style={styles.trackWidth}>
+                  <TD>{t('track')}</TD>
+                </View>
+                <View style={styles.endWidth}>
+                  <TD>{t('endStop')}</TD>
+                </View>
+                <View style={styles.passageWidth}>
+                  <TD>{t('passageStop')}</TD>
+                </View>
+                <View style={styles.startWidth}>
+                  <TD>{t('startStop')}</TD>
+                </View>
+                <View style={styles.weightWidth}>
+                  <TD>{t('weight')}</TD>
+                </View>
+                <View style={styles.refEngineWidth}>
+                  <TD>{t('referenceEngine')}</TD>
+                </View>
+                <View style={styles.convSignWidth}>
+                  <TD>{t('conventionalSign')}</TD>
+                </View>
+                <View style={styles.crossedATEWidth}>
+                  <TD>{t('crossedATE')}</TD>
+                </View>
+              </TH>
+              {stdcmData.simulation.base.stops.map((step, index) => {
+                const isFirstStep = index === 0;
+                const isLastStep = index === stdcmData.simulation.base.stops.length - 1;
+                const prevStep = stdcmData.simulation.base.stops[index - 1];
+                return (
+                  <TR
+                    key={index}
+                    style={step.duration !== 0 && !isLastStep ? styles.blueRow : styles.tbody}
+                  >
+                    <TD style={styles.indexColumn}>{index + 1}</TD>
+                    <View style={styles.opWidth}>
+                      <TD style={styles.td}>
+                        {!isFirstStep && !isLastStep && step.name === prevStep.name
+                          ? '='
+                          : step.name || 'Unknown'}
+                      </TD>
+                    </View>
+                    <View style={styles.chWidth}>
+                      <TD style={styles.chColumn}>{step.ch}</TD>
+                    </View>
+                    <View style={styles.trackWidth}>
+                      <TD style={styles.td}>{step.track_name}</TD>
+                    </View>
+                    <View style={styles.endWidth}>
+                      <TD style={styles.stopColumn}>
+                        {isLastStep || step.duration !== 0 ? getStopTime(step.time) : ''}
+                      </TD>
+                    </View>
+                    <View style={styles.passageWidth}>
+                      <TD
+                        style={{
+                          ...(step.duration !== 0 && !isLastStep
+                            ? {
+                                width: `${step.duration < 600 && step.duration > 60 ? 60 : 70}px`,
+                              }
+                            : {}),
+                          ...(step.duration !== 0 && !isLastStep
+                            ? styles.blueStop
+                            : styles.stopColumn),
```

Comment:
You check twice the same thing ;)
```suggestion
                          ...(step.duration !== 0 && !isLastStep
                            ? {
                                width: `${step.duration < 600 && step.duration > 60 ? 60 : 70}px`,
                                ...styles.blueStop,
                              }
                            : styles.stopColumn),
```

Reply:
done